packages/trader/src/App/Constants/header-links.js

```diff
@@ -1,12 +1,13 @@
-import React           from 'react';
-import { localize }    from 'App/i18n';
-import Icon            from 'Assets/icon.jsx';
-import { routes }      from 'Constants/index';
+import React        from 'react';
+import { localize } from 'App/i18n';
+import Icon         from 'Assets/icon.jsx';
+import IconDeriv    from 'Assets/Header/NavBar/icon-deriv.jsx';
+import { routes }   from 'Constants/index';
 
 const header_links = [
     {
         logo   : <div className='header__logo'>{localize('BETA')}</div>,
-        image  : <Icon icon='IconDeriv' className='header__icon' />,
+        image  : <IconDeriv className='header__icon' />,
```

Comment:
any reason to import the icon component directly? can we keep this as `<Icon icon='IconDeriv' ... />`?

Reply:
Because we don't want to LazyLoad the logo as it will have a non-smooth transition from skeleton in index.html to the react mounted `header.jsx`.

Reply:
okay thanks, means we do need to keep two types of icons (lazyloaded and regular ones) 🤔

Reply:
Yeah, it's just to make the transition more seamless in the case. 

The index.html is loading a skeleton with the icon loaded inline while `deriv-app` was previously loading the `header.jsx` with the LazyLoaded icon, so u'll see the Icon loaded in skeleton but suddenly it disappears and then reappears. (Before we removed the LazyLoading of the icon).